app/views/tasks/show.html.slim

```diff
@@ -504,23 +504,23 @@ legend.toggle-next
                   .row-value
                     = file.usage_by_lms
 .button-panel
-  /.button-box
+  .button-box
     .button-description
-      = "Save Exercises for later by adding them to your cart or a collection."
+      = "Save Tasks for later by adding them to your cart or a collection."
     .wrapper
-      = link_to t('exercises.show.add_to_cart'), add_to_cart_exercise_path(@task), method: 'post', class: 'btn btn-default'
+      /  = link_to t('exercises.show.add_to_cart'), add_to_cart_exercise_path(@task), method: 'post', class: 'btn btn-default'
       .dropdown.btn-group
         = button_tag class: 'btn btn-default dropdown-toggle', data: {toggle: 'dropdown'} do
-          = t('exercises.show.add_to_collection') + ' '
+          = t('tasks.show.add_to_collection') + ' '
```

Comment:
This is equal to (and for a better readability, we should probably switch):

```suggestion
          => t('tasks.show.add_to_collection')
```

[Read more here](https://github.com/slim-template/slim#output-).